checkin.js

```diff
@@ -177,7 +177,7 @@ const GLaDOSCheckIn = async () => {
 
             const PUSHPLUS = process.env.PUSHPLUS;
 
-            if (!PUSHPLUS.length) {
+            if (!PUSHPLUS || !PUSHPLUS.length) {

```

Comment:
这里应该只有 undefined 和 空字符串两种情况，其实 `!PUSHPLUS` 就可以。

```suggestion
            if (!PUSHPLUS) {

```

居然犯这种错误，不知道我当时怎么写的。